### PR TITLE
Provide signal of jacobian endeffector oriented to world frame

### LINKED ISSUE #41: Get jacobian rotated in the world frame
Right now the jacobians computed by the dynamic-pinocchio entity are rotated in the local frame of the frame they are computed at. Is there a way to get the jacobians at the frame location but oriented wrt. the world coordinate axis? This is as our force measurements are specified in world coordinates and therefore there must be a rotation to compute the effect of $J.T * \lambda$ in the dynamics equation.

### PR DESCRIPTION
**Pull request by jviereck**
Closes #41.

### CODE REVIEW

**Comment by jviereck** [comment]
ping @olivier-stasse, is there anything specific you want me to do for merging this PR?

**Comment by jviereck** [comment]
ping @nim65s, is there anything specific you want me to do for merging this PR?

**Comment by olivier-stasse** [comment]
Sorry I missed your messages.
I will treat your PR asap

**Comment by nim65s** [comment]
@jviereck : it would be better to make PR on the "devel" branch. 
Also, adding a test for this new feature would be great, but as there was not already one for the function you modified, I would understand if it was too much work.

File: include/sot-dynamic-pinocchio/dynamic-pinocchio.h [hunk 1]
```diff
@@ -103,7 +103,7 @@ class SOTDYNAMIC_EXPORT DynamicPinocchio
     public:
   /* --- SIGNAL ACTIVATION --- */
   dg::SignalTimeDependent< dg::Matrix,int >&
-    createEndeffJacobianSignal( const std::string& signame, const std::string& );
+    createEndeffJacobianSignal( const std::string& signame, const std::string&, const bool isLocal);
```
Comment (nim65s) [inline include/sot-dynamic-pinocchio/dynamic-pinocchio.h]: You can maybe use `const bool isLocal = true`, this way your modification will not risk to break anything relying on this API.

**Comment by jviereck** [comment]
@olivier-stasse: thanks for getting back on this PR!

@nim65s, thanks for your feedback. I've added the default parameter and pushed the change. Adding a unit test is more time consuming, given that there is no unit test for these functions yet and I would take your offer to land the PR as it is.

Let me know if there is anything else todo.

**Comment by olivier-stasse** [comment]
Validation using talos + SoT under gazebo

**Comment by jviereck** [comment]
Thanks for reviewing and merging this PR!

**Pull request merged**